rbac/management/audit_log/model.py

```diff
@@ -64,6 +64,7 @@ class AuditLog(TenantAwareModel):
 
     def get_tenant_id(self, request):
         """Retrieve tenant id from request."""
+        print("entering tenant")
```

Comment:
this is temporal ?

Reply:
Hello Libor! 
Yes! There's something from the IQE test that I want to test, but i no longer have access to the ephemeral env, so Im hoping that these print statements can help when vinicius runs the env!